UnitySDK/Assets/ML-Agents/Editor/BroadcastHubDrawer.cs

```diff
@@ -0,0 +1,204 @@
+    {
+        private BroadcastHub _hub;
+        // The height of a line in the Unity Inspectors
+        private const float LineHeight = 17f;
+        private const float ExtraSpaceBellow = 10f;
+        private const int ControlSize = 80;
+
+        /// <summary>
+        /// Computes the height of the Drawer depending on the property it is showing
+        /// </summary>
+        /// <param name="property">The property that is being drawn.</param>
+        /// <param name="label">The label of the property being drawn.</param>
+        /// <returns>The vertical space needed to draw the property.</returns>
+        public override float GetPropertyHeight(SerializedProperty property, GUIContent label)
+        {
+            LazyInitializeHub(property, label);
+            var numLines = _hub.Count + 2 + (_hub.Count > 0 ? 1 : 0);
+            return (numLines) * LineHeight + ExtraSpaceBellow;
+        }
+
+        /// <inheritdoc />
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            LazyInitializeHub(property, label);
+            position.height = LineHeight;
+            EditorGUI.LabelField(position, new GUIContent(label.text, 
+                "The Broadcast Hub helps you define which Brains you want to expose to " +
+                "the external process"));
+            position.y += LineHeight;
+
+            EditorGUI.BeginProperty(position, label, property);
+
+            EditorGUI.indentLevel++;
+            DrawAddRemoveButtons(position);
+            position.y += LineHeight;
+            
+            // This is the labels for each columns
+            var brainWidth = position.width - ControlSize;
+            var brainRect = new Rect(
+                position.x, position.y, brainWidth, position.height);
+            var controlRect = new Rect(
+                position.x + brainWidth, position.y, ControlSize, position.height);
+            if (_hub.Count > 0)
+            {
+                EditorGUI.LabelField(brainRect, "Brains");
+                brainRect.y += LineHeight;
+                EditorGUI.LabelField(controlRect, "Control");
+                controlRect.y += LineHeight;
+                controlRect.x += 15;
+            }
+            DrawBrains(brainRect, controlRect);
+            EditorGUI.indentLevel--;
+            EditorGUI.EndProperty();
+        }
+        
+        /// <summary>
+        /// Draws the Add and Remove buttons.
+        /// </summary>
+        /// <param name="position">The position at which to draw.</param>
+        private void DrawAddRemoveButtons(Rect position)
+        {
+            // This is the rectangle for the Add button
+            var addButtonRect = position;
+            addButtonRect.x += 20;
+            if (_hub.Count > 0)
+            {
+                addButtonRect.width /= 2;
+                addButtonRect.width -= 24;
+                if (GUI.Button(addButtonRect, new GUIContent("Add New",
```

Comment:
for readability, create temporary variable for new GUIContent("Add New", "Add a new Brain to the Broadcast Hub")